### ISSUE TITLE
Salmon output files will be overwritten by other samples

### ISSUE THREAD

**Issue by kurtwheeler** [AlexsLemonade/refinebio#273]
### Context

The output directory for salmon jobs is on a per-experiment basis, however the processor jobs are run on a per-sample basis. This means that as each processor job finishes, it will overwrite the `quant.sf` file that was output by any jobs for that experiment which already finished.

### Problem or idea

We need to create per-sample or per-job output directories. The tricky part to this is making them discoverable from either ComputationalResult or ComputedFile objects.

### Solution or next step

Create separate directories for each salmon job so that files won't conflict. Find a way to track these down from database objects.

### New Issue Checklist

- [x] The title is short and descriptive.
- [x] You have explained the context that led you to write this issue.
- [x] You have reported a problem or idea.
- [x] You have proposed a solution or next step.

**Comment by dongbohu** [comment]
Are you saying that  the output files were saved in this directory? 
```<experiment_name>/processed/``` 

I thought they were in: 
```<experiment_name>/<sample_name>/processed/```

**Comment by kurtwheeler** [comment]
Sorry, I wanted to run a test to verify this and I forgot to come back and actually answer this question. `<experiment_name>/<sample_name>/processed/` does not seem to be the case:

```
$ ls volume/ERP016796/processed/
aux_info  cmd_info.json  lib_format_counts.json  libParams  logs  quant.sf
```

**Comment by dongbohu** [comment]
Never mind. I confirmed this from Jackie yesterday. So I deleted my question. I am going to fix it in the upcoming PR.

**Comment by kurtwheeler** [comment]
Oh sorry, I hadn't refreshed the page. Awesome that you're gonna fix it though!

**Comment by dongbohu** [comment]
Fixed in PR #278.
